scripts/download-extensions

```diff
@@ -64,7 +73,9 @@ get_rtheaders() {
     info "Retrieving ${headers} for ${rtrpm}"
     yumdownload "${headers}"
     headersrpm=$(ls kernel-headers*)
-    ln "${headersrpm}" ../..
+    if [ ! -f "${destdir}/extensions/${headersrpm}" ]; then
+        ln "${headersrpm}" "${destdir}/extensions/"
+    fi
```

Comment:
Hmm, I don't think we actually need this since we're always creating the `kernel-rt` extension first, so we should never hit this case. But WFM as is.